Makefile

```diff
@@ -19,7 +20,7 @@ dir:
 	mkdir -p $(OUT_DIR)
 
 $(PROJNAME).pdf: $(PROJNAME).tex
-	latexmk -outdir=$(OUT_DIR) $<
+	latexmk -lualatex -outdir=$(OUT_DIR) $<
```

Comment:
It should be possible to replace the option `-lualatex`, by adding  `$pdf_mode = 4;` to the configuration file

According to this [documentation](https://ftp.rrzn.uni-hannover.de/pub/mirror/tex-archive/support/latexmk/latexmk.pdf), it should do the same:

> If equal to 4, generate a pdf version of the document using lualatex, using the command specified by the $lualatex variable

Reply:
Yeah, `$pdf_mode = 4;` works. Moving all the configuration to the .latexmkrc file is way more elegant. I will update my pull request.